client/app/hosting/configuration/CONFIGURATION.html

```diff
@@ -0,0 +1,42 @@
+<div class="container-fluid px-0" data-ng-controller="HostingTabConfigurationCtrl as ctrl">
+    <div class="row">
+        <div class="col-md-9">
+            <div data-ovh-alert="{{ alerts.main }}"></div>
+
+            <oui-spinner data-size="l" data-ng-if="ctrl.loading.init"></oui-spinner>
+            <div data-ng-if="!ctrl.loading.init && ctrl.hasResult">
+                <div data-ng-include="ctrl.templates.runtimeList"></div>
+                <br><br>
+
+                <div data-ng-include="ctrl.templates.envvarList"></div>
+            </div>
+
+            <div data-ng-if="!ctrl.loading.init && !ctrl.hasResult">
+                <div class="alert alert-info mt-5" role="alert">
+                    <div data-i18n-static="hosting_tab_CONFIGURATION_empty"></div>
+                </div>
+            </div>
+        </div>
+
+        <aside class="col-md-3 mt-5 mt-lg-0">
+            <div class="mb-5">
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_runtime_add}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_runtime_add"
+                        data-ng-click="setAction('configuration/runtime/add-or-edit/hosting-configuration-runtime-add-or-edit', {})"
+                        ng-if="hosting.offer !== 'CLOUDWEBBETAX_1' || (hosting.offer === 'CLOUDWEBBETAX_1' && ctrl.runtimes.ids.length < 1)">
+                </button>
+
+                <button type="button" class="btn btn-block btn-default text-wrap" title="{{i18n.hosting_tab_CONFIGURATION_runtime_quota_reached}}"
+                        data-i18n-static="hosting_tab_CONFIGURATION_runtime_add"
+                        ng-if="hosting.offer === 'CLOUDWEBBETAX_1' && ctrl.runtimes.ids.length >= 1"
```

Comment:
why did you not use capabilities from  /hosting/web/offerCapabilities to enable, or not, runtimes?

Reply:
Because I forgot !